internal/pkg/workspace/workspace_test.go

```diff
@@ -1559,8 +1559,8 @@ func TestWorkspace_DeleteWorkspaceFile(t *testing.T) {
 					Fs: fs,
 				},
 			}
-			ws.fs.MkdirAll("copilot", 0755)
-			ws.fs.Create(tc.copilotDir + "/" + ".workspace")
+			ws.fs.MkdirAll(tc.copilotDir, 0755)
+			ws.fs.Create(filepath.Join(tc.copilotDir, ".workspace"))
```

Comment:
do you know if we really need these? They seem to be doing the same as as in `tc.fs()` 🤔